extensions/amp-story/1.0/story-analytics.js

```diff
@@ -37,6 +37,7 @@ export const StoryAnalyticsEvent = {
   PAGE_ATTACHMENT_ENTER: 'story-page-attachment-enter',
   PAGE_ATTACHMENT_EXIT: 'story-page-attachment-exit',
   PAGE_VISIBLE: 'story-page-visible',
+  INTERACTIVE_ELEMENT_RESPOND: 'interactive-element-respond',
```

Comment:
Triggers that we want to be handled by `amp-story` and not general AMP components should start with the prefix `story-*`, this ensures that the `AmpStoryEventTracker` will handle the logic, which we want in this case.

Reply:
I'm thinking we might be able to reuse the `story-click-through` trigger and have publishers specify a configuration like this one:

```
              "trackQuizClick": {
                "on": "story-click-through",
                "tagName": "interactive-element-respond", // or whatever this.quizEl_[ANALYTICS_TAG_NAME] is
                "request": "click"
              },
```
This way we aren't introducing a new trigger.

Reply:
Oh good simplification, I'll modify to reusing that trigger